main/main.cpp

```diff
@@ -4051,7 +4056,8 @@ int Main::start() {
 				sml->get_root()->set_embedding_subwindows(true);
 			}
 			restore_editor_window_layout = EditorSettings::get_singleton()->get_setting(
-					"interface/editor/remember_window_size_and_position");
+																				  "interface/editor/editor_screen")
+												   .operator int() == DisplayServer::SCREEN_AUTO;
```

Comment:
This formatting looks terrible.
It would be more readable as a single line.

Reply:
😅  Battling with auto formatting on this one